tests/Resources/ti.filesystem.file.test.js

```diff
@@ -846,87 +833,135 @@ describe('Titanium.Filesystem.File', function () {
 		it('file:// absolute path', () => {
 			const file = Ti.Filesystem.getFile(fileURI);
 			should.exist(file);
-			should(file.exists()).equal(true); // FIXME: Fails on Android, but only if run as part of full suite!
+			should(file.exists()).be.true(); // FIXME: Fails on Android, but only if run as part of full suite!
 		});
 
 		it.androidAndIosBroken('file:// relative path', () => {
 			// FIXME: Android seems to basically forcibly place '/' in front of paths not beginning with '..' or '/'
 			// FIXME: iOS does not seem to try and resolve relative paths for file:// URIs
 			const file = Ti.Filesystem.getFile('file://app.js'); // app.js should be relative to this file...
 			should.exist(file);
-			should(file.exists()).equal(true);
+			should(file.exists()).be.true();
```

Comment:
NOTE: Should these tests be enabled on Android now?

Reply:
This test still won't work on Android. We would have to modify `TiFileFactory` to check if the "file://" URL has a relative and have it use our `TiResourceFile` instead of a `TiFile` object since our APK's "assets/Resources" folder is not accessible from the file system.

But the problem here is if you set a relative path without a URL scheme like this...
```javascript
Ti.Filesystem.getFile('app.js');
```
...then `TiFileFactory` defaults to the `applicationDataDirectory`, not `resourcesDirectory` like this unit test assumes. So, we have an inconsistency here.